frontend/index.js

```diff
@@ -1,24 +1,18 @@
-// Temporary server for Azure Dev
-const BASE_PATH = "./";
+const express = require('express');
+const path = require('path');
 
-Bun.serve({
-    port: 3000,
-    async fetch(req) {
-        const url = new URL(req.url);
-        let filePath = BASE_PATH + url.pathname;
+const app = express();
```

Comment:
@tdonaworth - Any thoughts on this one? I can definitely do something in the express server in the container but then the app logic will need to do validation I believe?

Reply:
@devops-chris - Are you proposing moving from `Bun.serve` to just using Express? If so, then yea we'd need to put something in place 🤔 Need to think about this.

Reply:
I moved from Bun.serve to Express because Bun wasn't handling the condition for catching all non-routable traffic correctly. At least I couldn't get it to work after trying repeatedly. However, I don't think that is the issue here... I just think the alert showed up because it is express, but the issue of XSRF would be a thing no matter how we serve it, right? We can handle it with Express though.